python/lsst/meas/deblender/baseline.py

```diff
@@ -112,24 +111,19 @@ def __init__(self, footprint, maskedImages, psfs, psffwhms, log, filters=None,
                                                             avgNoise]]))
 
         self.log = log
-        self.filterCount = len(maskedImages)
-        self.maskedImages = maskedImages
+        self.mMaskedImage = mMaskedImage
         self.footprint = footprint
         self.psfs = psfs
 
         self.peakCount = len(footprint.getPeaks())
         if maxNumberOfPeaks>0 and maxNumberOfPeaks<self.peakCount:
             self.peakCount = maxNumberOfPeaks
 
-        if filters is None:
-            filters = range(self.filterCount)
-        self.filters = filters
-
         # Create a DeblendedParent for the Footprint in every filter
         self.deblendedParents = OrderedDict([])
-        for n in range(self.filterCount):
+        for n,f in enumerate(self.filters):
```

Comment:
Space after comma?

Reply:
Perhaps use `n, filter` as below?

Reply:
The problem with filter is that it's a builtin. I know that some people use it in the stack but I prefer to just use `f`, although if it really bugs you I can change the name.